dbms/src/Common/getNumberOfCPUCores.cpp

```diff
@@ -46,6 +46,8 @@ void computeAndSetNumberOfPhysicalCPUCores(
     UInt16 number_of_logical_cpu_cores_,
     UInt16 number_of_hardware_physical_cores)
 {
+    number_of_hardware_physical_cores = std::max<UInt16>(1, number_of_hardware_physical_cores);
```

Comment:
Could you please describe what're the acutual parameter values when the issue happened?

Reply:
> Could you please describe what're the acutual parameter values when the issue happened?

When the issue happens, hardware_logical_cpu_cores is lower than number_of_hardware_physical_cores which causes (hardware_logical_cpu_cores / number_of_hardware_physical_cores) is equal to 0.